core/block/import/markdown/anymark/markdown.go

```diff
@@ -233,6 +230,19 @@ func getCustomHTMLRules() []html2md.Rule {
 		simpleText, blockquote, italic, code, bdo, div, img, table}
 }
 
+func extractImageSource(selec *goquery.Selection) string {
+	var (
+		src string
+		ok  bool
+	)
```

Comment:
These declarations are redundunt as we initialize both of these variables in line 238

Reply:
We need it, because we return src in the end of the function